tests/phpunit/tests/block-template-utils.php

```diff
@@ -412,7 +412,7 @@ public function test_wp_generate_block_templates_export_file() {
 	public function test_inject_ignored_hooked_blocks_metadata_attributes_into_template() {
 		global $wp_current_filter;
 		// Mock currently set filter. The $wp_current_filter global is reset during teardown by
-		// WP_UnitTestCase_Base::_restore_hooks() in tests/phpunit/includes/abstract-testcase.php.
+		// the unit test base class.
```

Comment:
This is in response to [this comment](https://core.trac.wordpress.org/ticket/60671#comment:18):

> Nit for [[57799]](https://core.trac.wordpress.org/changeset/57799): don't see a reason for mentioning the `tests/phpunit/includes/abstract-testcase.php` file path in the new comment there. That will just become stale with any refactor, and any IDE will allow you to quickly jump to the definition. But it is what it is :)